funcx/executors/high_throughput/executor.py

```diff
@@ -29,8 +31,10 @@
 
 
 from funcx.executors.high_throughput import zmq_pipes
+from funcx import set_file_logger
 
-logger = logging.getLogger(__name__)
+# logger = logging.getLogger(__name__)
+logger = set_file_logger("executor.log", name=__name__)
```

Comment:
We could use something like
```python
logger = logging.getLogger(__name__)
if not logger.hasHandlers():
    logger = set_file_logger("executor.log", name=__name__)
```
Is that appropriate?  Could also just remove this.  It did make it much easier to debug and was hard to tell where logs were going to route to otherwise.

Reply:
This makes sense. Let's use a logger if one is available.